tests/test_enterprise/api/test_serializers.py

```diff
@@ -337,3 +338,8 @@ def test_create(self):
         self.data['pgp_encryption_key'] = 'invalid-key'
         serializer = EnterpriseCustomerReportingConfigurationSerializer(data=self.data)
         assert not serializer.is_valid()
+
+        # Valid Compression check should be flagged.
+        self.data['enable_compression'] = False
+        serializer = EnterpriseCustomerReportingConfigurationSerializer(data=self.data)
+        assert not serializer.is_valid()
```

Comment:
Check error message here, It should have expected error message.

Reply:
Good idea to make it more efficient.